src/main/java/org/opensearch/agent/tools/RAGTool.java

```diff
@@ -71,6 +71,11 @@ public class RAGTool implements Tool {
     @Setter
     private Parser outputParser;
 
+    @Override
+    public boolean useOriginalInput() {
```

Comment:
Thanks @SuZhou-Joe to help flipping on the flag following the fix in common https://github.com/opensearch-project/ml-commons/pull/2022 using original input for RAGTool/VectorDBTool/NeuralSparseSearchTool. IMO, the neuralSparseSearchTool also need to take original input question instead of llm generated questions. @zhichao-aws would you agree on this for NeuralSparseSearchTool?

In this case, since vectorDB and NeuralSparseSearchTool are both extending from AbstractRetrieverTool, we can modify the AbstractRetrieverTool to flip on UseOriginalInput to true. Then both VectorDB and NeuralSparseSearchTool are using original input. 

RAGTool also need to keep this useOrginalInput flag because it's not extending from AbstractRetrieverTool now.